tests/main.test.js

```diff
@@ -69,3 +69,68 @@ test('media queries print removed', async () => {
   const { finalCss } = await runMinimalcss('media-queries-print')
   expect(finalCss).toEqual(result)
 })
+
+test('removes unused @keyframes', async () => {
+  const result = ''
+  const { finalCss } = await runMinimalcss('keyframe-removes')
+  expect(finalCss).toEqual(result)
+})
+
+test('leaves used @keyframes', async () => {
+  const result =
+    '@keyframes RotateSlot{3%{margin-top:-2em}0%{transform:rotate(0deg)}}.SomeSelector{animation:RotateSlot infinite 5s linear}'
+  const { finalCss } = await runMinimalcss('keyframe-leaves')
+  expect(finalCss).toEqual(result)
+})
+
+test('removes used inline @keyframes', async () => {
+  const result = ''
+  const { finalCss } = await runMinimalcss('keyframe-removes-inline')
+  expect(finalCss).toEqual(result)
+})
+
+test('removes unused @fontface', async () => {
+  const result = ''
+  const { finalCss } = await runMinimalcss('fontface-removes')
+  expect(finalCss).toEqual(result)
+})
+
+test('leaves used @fontface', async () => {
+  const result =
+    "@font-face{font-family:'Lato';font-style:normal;font-weight:400;src:local('Lato Regular'),local('Lato-Regular'),url(https://fonts.gstatic.com/s/lato/v14/MDadn8DQ_3oT6kvnUq_2r_esZW2xOQ-xsNqO47m55DA.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2212,U+2215}.SomeSelector{font-family:'Lato'}"
+  const { finalCss } = await runMinimalcss('fontface-leaves')
+  expect(finalCss).toEqual(result)
```

Comment:
Can't you do something like `expect(finalCss).toContain("@font-face{font-family:'Lato'")`?